terralab/commands/pipeline_runs_commands.py

```diff
@@ -145,26 +145,22 @@ def list_command(num_results: int) -> None:
         row_list = [
             [
                 "Job ID",
-                "Pipeline Name",
-                "Version",
+                "Pipeline",
                 "Status",
                 "Submitted",
-                "Completed",
+                "Output Expiration Date",
                 "Description",
-                "Output Expiration Date"
             ]
         ]
         for pipeline_run in results:
             row_list.append(
                 [
                     pipeline_run.job_id,
-                    pipeline_run.pipeline_name,
-                    str(pipeline_run.pipeline_version),
+                    f"{pipeline_run.pipeline_name}{f' v{pipeline_run.pipeline_version}' if pipeline_run.pipeline_version else ''}",
```

Comment:
hmm pipeline version should never be null so we shouldn't need this statement (like id rather it fail if it was null so we can fix the issue)